[Feature] [LOR 349/351]  Navigation FAQ and  Terms the MenuView.

sucriss2 opened the pull request:
### Descrição 
✅ _Navegação para Perguntas frequentes._
✅ _Navegação para Termos e Privacidade._
✅ _Mudança no frame da HomeView, nao aceita (width: .infinity)_
✅ _Correções do lint._
✅ _Mudança de cor e font dos textos na tela de menu._
✅ _Mudança de imagem da logo na tela de Home._
✅ _Alteração de layout frase de boas vindas da Home._

### Evidências
| 📱  EVIDÊNCIA  - Iphone (iPhone SE (3rd generation) - iOS 17.4 ) |
| :------------------------------: |
| <kbd>![Simulator Screen Recording - iPhone SE (3rd generation) - 2024-10-23 at 15 32 11](https://github.com/user-attachments/assets/e9ea9baa-9070-4dd7-a8e9-e7ad571d0513)</kbd>|


### Evidências
| 📱  Sem alteração  - Iphone (iPhone SE (3rd generation) ) |
| :------------------------------: |
| <img width="218" alt="Captura de Tela 2024-10-24 às 19 05 14" src="https://github.com/user-attachments/assets/6cf5cf1f-17ed-4fdf-a7d5-06644dc697ef">
|

### Evidências
| 📱  Com alteração  - Iphone (iPhone SE (3rd generation) ) |
| :------------------------------: |
| <img width="217" alt="Captura de Tela 2024-10-24 às 19 02 43" src="https://github.com/user-attachments/assets/bc3169f9-5310-44b7-b5a6-9dea663cb5bf">|

sucriss2 committed:
Crate new view to webview terms and faq.
- Corretion the switlint, lines and methods.
- Change frame the HomeView.

sucriss2 committed:
Corretion lint

dedeexe commented on loryblu/loryblu/Feature/Menu/MenuWebView.swift:
```diff
@@ -0,0 +1,40 @@
+import SwiftUI
+
+struct MenuWebView: View {
```
@sucriss2 
Que tal criar um modifier que adiciona o título + o botão fechar na tela? Assim este esquema de título + fechar fica reaproveitável.

Toda e qualquer tela que quiser utilizar este mesmo recurso poderia utilizar-lo com poucas linhas de código :)

sucriss2 replied:
Boa ideia, vou deixar anotado pra melhoria e subo como uma feature/improvement, e marco vcs no discord.

rodrigoliveirac commented on loryblu/loryblu/Feature/Menu/MenuViewModel.swift:
```diff
@@ -1,16 +1,20 @@
 import Foundation
+import SwiftUI
```
Nós só precisamos desse import quando estamos escrevemos views.

sucriss2 replied:
Esqueci de excluir. 🙃

rodrigoliveirac commented on loryblu/loryblu/Feature/Menu/MenuView.swift:
```diff
@@ -4,6 +4,12 @@ struct MenuView: View {
+    @State var urlString = String()
+    @State var urlFaq = Server.faq
+    @State var urlTerms = Server.termsOfUse
+    @State private var navigationTitle = String()
```
O que acha de padronizarmos todos esses estados atribuídos ao MenuViewModel?

rodrigoliveirac commented on loryblu/loryblu/Feature/Menu/MenuView.swift:
```diff
@@ -53,25 +62,42 @@ struct MenuView: View {
+                            onClick: {
+                                urlString = urlTerms
+                                navigationTitle = LBStrings.Menu.privacyTerms
+                                showWebView = true
```
Nesse escopo, por exemplo, poderíamos chamar apenas uma função na MenuViewModel e atualizar seus atributos conforme. Assim, mantemos a lógica da UI isolada na ViewModel.

sucriss2 replied:
Show demais.

rodrigoliveirac reviewed (changes requested):
Está ótimo pra mim! Deixei alguns comentários importantes.

sucriss2 committed:
Change image logo homeView.
- Change font and color the texts screen MenuView.
- Correction comment in PR, change method and delete vars.
- Change corretion lint.

Rodrigo-Eduardo-Silva commented on loryblu/loryblu/Feature/Home/HomeView.swift:
```diff
@@ -96,7 +105,7 @@ struct HomeView: View {
                     }
                 }
-                .frame(width: .infinity, height: 218)
+                .frame(minWidth: 324, minHeight: 218)
```
Tem que deixar com .infinity , se não teremos o mesmo problema o qual você pediu bug fix na homeview.

sucriss2 replied:
Vou modificar aqui pra nao deixar o tamanho fixo, mas nao pode usar mais o .infinity no width, tem que ser ou no minWidth ou maxWidth. Mas alterei aqui.

sucriss2 committed:
Change image HomeView.
-Change contranst home.
